nucypher/characters/control/interfaces.py

```diff
@@ -96,7 +97,8 @@ def grant(self,
 
         response_data = {'treasure_map': new_policy.treasure_map,
                          'policy_encrypting_key': new_policy.public_key,
-                         'alice_verifying_key': new_policy.alice_verifying_key}
+                         # TODO: technically we should return both Alice's and Publisher's keys here.
```

Comment:
If not being addressed in the PR, does this need a follow-up issue?

Reply:
I am actually not sure now if the interface warrants it. Can a policy with Provider and Publisher being different characters be even created at the moment, in any way?

Reply:
that's a question for @KPrasch I guess...

Reply:
After some thought, I think it's justified to return only the Alice's key - currently for this interface there's no way to have a separate publisher. When this capability is added, we can extend the return value.